content/applications/hr/referrals.rst

```diff
@@ -656,8 +502,10 @@ If desired, the :guilabel:`Image` can also be modified. Hover over the image to
 (pencil)` icon and a file explorer window appears. Navigate to the desired image file then click
 :guilabel:`Open` to select it.
 
-Once all changes have been made, click the :guilabel:`Save` button to save the changes, or click the
-:guilabel:`Discard` button to revert to the original information.
+The level form saves automatically but can be saved manually at any time by clicking the *save
+manually* option, represented by a :guilabel:`(cloud upload)` icon, located in the top-left corner.
+To cancel any changes made, click the :guilabel:`✖️ (Discard all changes)` icon to delete any
```

Comment:
is it an 'X' or a 'counter-clockwise arrow'?

Reply:
Yup- it's an X